<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1367</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="PaperList" name="listWidget"/>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="newPaperButton">
      <property name="text">
       <string>New Paper</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" rowspan="3">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="MPBrushSelector" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1367</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelect"/>
   <addaction name="actionDraw"/>
   <addaction name="actionText"/>
   <addaction name="separator"/>
   <addaction name="actionBlack"/>
   <addaction name="actionBlue"/>
   <addaction name="actionGreen"/>
   <addaction name="actionRed"/>
   <addaction name="separator"/>
   <addaction name="actionballpen"/>
   <addaction name="actionAcrylic"/>
   <addaction name="actionPencil"/>
   <addaction name="actioncalligraphy"/>
   <addaction name="actionsewing"/>
   <addaction name="actionwaterColor"/>
   <addaction name="separator"/>
   <addaction name="actionEraser"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDraw">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/assets/pencil.svg</normaloff>:/assets/pencil.svg</iconset>
   </property>
   <property name="text">
    <string>Draw</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionText">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/assets/text_tool_icon.svg</normaloff>:/assets/text_tool_icon.svg</iconset>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
   <property name="toolTip">
    <string>Insert Text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/assets/cursor.svg</normaloff>:/assets/cursor.svg</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="toolTip">
    <string>Select Mode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S, Esc</string>
   </property>
  </action>
  <action name="actionBlack">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/assets/black.png</normaloff>:/assets/black.png</iconset>
   </property>
   <property name="text">
    <string>Black</string>
   </property>
  </action>
  <action name="actionBlue">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/assets/blue.png</normaloff>:/assets/blue.png</iconset>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
  </action>
  <action name="actionGreen">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/assets/green.png</normaloff>:/assets/green.png</iconset>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
  </action>
  <action name="actionRed">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/assets/red.png</normaloff>:/assets/red.png</iconset>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </action>
  <action name="actionballpen">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/brushes/deevad/ballpen_prev.png</normaloff>:/brushes/deevad/ballpen_prev.png</iconset>
   </property>
   <property name="text">
    <string>ballpen</string>
   </property>
  </action>
  <action name="actionAcrylic">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/brushes/tanda/acrylic-05-with-water_prev.png</normaloff>:/brushes/tanda/acrylic-05-with-water_prev.png</iconset>
   </property>
   <property name="text">
    <string>Acrylic</string>
   </property>
  </action>
  <action name="actionPencil">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/brushes/deevad/2B_pencil_prev.png</normaloff>:/brushes/deevad/2B_pencil_prev.png</iconset>
   </property>
   <property name="text">
    <string>Pencil</string>
   </property>
  </action>
  <action name="actionwaterColor">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/brushes/deevad/watercolor_glazing_prev.png</normaloff>:/brushes/deevad/watercolor_glazing_prev.png</iconset>
   </property>
   <property name="text">
    <string>waterColor</string>
   </property>
  </action>
  <action name="actioncalligraphy">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/brushes/classic/calligraphy_prev.png</normaloff>:/brushes/classic/calligraphy_prev.png</iconset>
   </property>
   <property name="text">
    <string>calligraphy</string>
   </property>
  </action>
  <action name="actionsewing">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/brushes/experimental/sewing_prev.png</normaloff>:/brushes/experimental/sewing_prev.png</iconset>
   </property>
   <property name="text">
    <string>sewing</string>
   </property>
  </action>
  <action name="actionEraser">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/brushes/deevad/large_hard_eraser_prev.png</normaloff>:/brushes/deevad/large_hard_eraser_prev.png</iconset>
   </property>
   <property name="text">
    <string>Eraser</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PaperList</class>
   <extends>QListWidget</extends>
   <header>paperlist.h</header>
  </customwidget>
  <customwidget>
   <class>MPBrushSelector</class>
   <extends>QTabWidget</extends>
   <header>mpbrushselector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
